homeassistant/components/rfxtrx.py

```diff
@@ -28,8 +28,14 @@
 ATTR_FIREEVENT = 'fire_event'
 ATTR_DATA_TYPE = 'data_type'
 ATTR_DUMMY = 'dummy'
+ATTR_OFF_DELAY = 'off_delay'
+ATTR_SENSOR_CLASS = 'sensor_class'
```

Comment:
Some of this constants can be found in homeassistant.const
 from homeassistant.const import CONF_SENSOR_CLASS